pkg/manifestgen/install/install.go

```diff
@@ -116,10 +116,6 @@ func GetLatestVersion() (string, error) {
 
 // ExistingVersion calls the GitHub API to confirm the given version does exist.
 func ExistingVersion(version string) (bool, error) {
-	if !strings.HasPrefix(version, "v") {
```

Comment:
We can't remove this from here as it breaks https://github.com/fluxcd/terraform-provider-flux